### ISSUE TITLE
Tiny guide for NPM and VSC?

### ISSUE THREAD

**Issue by IlluminatiWave** [frida/frida-node#94]
I wanted to install the NPM version to be able to access the Frida syntax in a typescript script, but I had problems.

1. There is no way to run the module from the global environment (npm install -g frida).

2. Install it in the “project” (I had to create a new folder just for the script and convert it to vsc project) it detects it, but it doesn't find the syntax.

I tried using

import * as frida from 'frida';

but it tells me that 

The property 'Process' does not exist in type 'typeof import(“d:/New folder (2)/node_modules/frida/dist/index”)'.ts(2339)

in short, I cannot access the simple

const module = Process.findModuleByName(moduleName);

because it does not exist, either process or findModuleByName inside the npm module.

PS D:\New folder (2)> npm list --depth=0   
New folder (2)@ D:\New folder (2)
├── frida@16.3.3
└── typescript@5.4.5
